docs/user/docs/settings/tools/capella/t4c-git-compare.md

```diff
@@ -0,0 +1,77 @@
+<!--
+ ~ SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+ ~ SPDX-License-Identifier: Apache-2.0
+ -->
+
+# Ways of co-working on a Capella project
+
+## Quick comparision
+
+<table markdown="1">
+  <tr>
+    <th></th>
+    <th>Git</th>
+    <th>TeamForCapella</th>
+  </tr>
+  <tr>
+    <td><b>Multiple workstreams</b></td>
+    <td>Different workstreams available via branches</td>
+    <td>Not recommended, only possible via different repositories</td>
+  </tr>
+  <tr>
+    <td><b>Concurrent working</b></td>
+    <td>Very challenging because of merge conflict potential.</td>
+    <td>Is possible. Diagrams are locked, no risk of merge conflicts.</td>
+  </tr>
+  <tr>
+    <td><b>Merge conflicts</b></td>
+    <td>Resolution of merge conflicts can be challenging.</td>
+    <td>No merge conflicts when one workstream is used. For multiple workstreams, the resolution of merge conflicts can be challenging.</td>
+  </tr>
+  <tr markdown="1">
+  <!-- This formatting is required, otherwise mkdocs breaks -->
+  <td><b>License</b></td>
+  <td markdown="1">No license needed[^1]</td>
+  <td markdown="1">TeamForCapella license required[^2]</td>
+  </tr>
+  <tr>
+    <td><b>Complexity</b></td>
+    <td>High complexity, Git knowledge is required.</td>
+    <td>Easy to use</td>
+  </tr>
+  <tr>
+    <td><b>Stability</b></td>
+    <td>Very stable solution as changes are initially saved locally. Temporary server downtimes can also be bridged.</td>
+    <td>Changes may be lost if the server becomes unavailable during work.</td>
+  </tr>
+  <tr>
+    <td><b>Potential of data loss</b></td>
+    <td>Changes are backed up after each push (more regularly).</td>
+    <td>Changes are backed up on a nightly basis.</td>
+  </tr>
+  <tr>
+    <td><b>Automation</b></td>
+    <td>Possible via CI/CD in the Git repository.</td>
+    <td>Possible via TeamForCapella &rarr; Git synchronisation (runs on a nightly basis).</td>
+  </tr>
+  <tr>
+    <td><b>Change control</b></td>
+    <td>Reviews are possible via merge requests.</td>
+    <td>Change control is nearly impossible, no barriers.</td>
+  </tr>
+  <tr>
+    <td><b>Release tagging</b></td>
+    <td>Directly possible via tags in Git.</td>
+    <td>Possible via TeamForCapella &rarr; Git synchronisation (Releases are stored as tags in the Git repository).</td>
```

Comment:
what is meant by "the" Git repository - are we not talking about TeamForCapella here?

Reply:
This is possible via the `TeamForCapella` > `Git` synchronization. The content is synchronized on a nightly basis from T4C to Git, so tags can still be created in the Git repository.